webapp/channels/src/components/advanced_create_comment/advanced_create_comment.tsx

```diff
@@ -438,7 +443,7 @@ class AdvancedCreateComment extends React.PureComponent<Props, State> {
 
         const hasSelection = !isNil(selectionStart) && !isNil(selectionEnd) && selectionStart < selectionEnd;
         const hasTextUrl = isTextUrl(clipboardData);
-        const hasHTMLLinks = hasHtmlLink(clipboardData);
+        const hasHTMLLinks = !this.isNonFormattedPaste && hasHtmlLink(clipboardData);
```

Comment:
Is there any chance of race condition here? Or we are positive that the pastHandler will be called after the keydown event always (and within 250ms)?

Reply:
yes, you are right, paste is supposed and normally called after keydown in the JS series of events, however, I added the extra 250ms as an extra assurement method.